themes/tdr/css-src/sass/_header.scss

```diff
@@ -19,7 +19,84 @@ $govuk-header-link: govuk-colour('white');
     padding-inline-start: 0;
   }
 
-  .govuk-header__link--service-name {
+  .govuk-header__link--service-name--tdr {
     margin-bottom: 0;
   }
 }
+
+@mixin on-mobile {
+  @media screen and (width <=768px) {
+    @content;
+  }
+}
+
+.tna-header__link--homepage {
+  &:hover,
+  &:active {
+    border-bottom: none;
+    text-decoration: none;
+    margin-bottom: 0.8rem;
+  }
+
+  @include on-mobile {
+    margin-right: 0;
+    font-size: 0.8rem;
+    display: inline-block;
+  }
+}
+
+.tna-header {
```

Comment:
@TomJKing Is .tna ok as a prefix for these classes. If anything is preferred we can use that instead